jira/server/scio_search/src/main/java/ScioSearchConfigRestPlugin/impl/ScioSearchIssueSecurityMembers.java

```diff
@@ -0,0 +1,134 @@
+        logger.debug(isl.getDescription() + ":" + isl.getName() + ":" + String.valueOf(isl.getId()) + ":" + String.valueOf(isl.getSchemeId()));
+        List<IssueSecurityLevelPermission> issueSecurityLevelPermissions = issueSecuritySchemeManager.getPermissionsBySecurityLevel(isl.getId());
+        if (issueSecurityLevels != null) {
+          response.maxResults += 1;
+          response.total += 1;
+          response.values = new ArrayList();
+          for (IssueSecurityLevelPermission islp : issueSecurityLevelPermissions) {
+            logger.debug(islp.getId() + ":" + islp.getParameter() + ":" + islp.getSchemeId() + ":" + islp.getSecurityLevelId() + ":" + islp.getType());
+            IssueSecuritySchemeMemberInfo memberInfo = new IssueSecuritySchemeMemberInfo();
+            memberInfo.id = String.valueOf(islp.getId());
+            memberInfo.issueSecurityLevelId = String.valueOf(islp.getSecurityLevelId());
+            JiraPermissionHolderInfo holder = new JiraPermissionHolderInfo();
+            holder.type = islp.getType();
+            memberInfo.holder = holder;
+            switch (islp.getType()) {
+              case "applicationRole":
+                break;
+              case "user":
+                // TODO: Set accountId or name? Set active based on api call?
+                holder.user = new AtlassianUser();
+                holder.user.key = islp.getParameter();
+                holder.user.active = true;
+                break;
+              case "group":
+                // TODO: Handle deleted group issue.
+                holder.group = new AtlassianGroup();
+                holder.group.name = islp.getParameter();
+                break;
+              case "projectrole":
+                holder.type = "projectRole";            // Overwrite the type since the Jira Cloud response has this.
+                holder.parameter = islp.getParameter(); // roleId is the parameter.
+                break;
+              case "lead":
+                holder.type = "projectLead";            // Overwrite the type since the Jira Cloud response has this.
+                break;
+              case "reporter":
+                break;
+              case "assignee":
+                break;
+              case "userCF":
+                holder.type = "userCustomField";        // Overwrite the type since the Jira Cloud response has this.
+                holder.parameter = islp.getParameter(); // custom field name
+                break;
+              case "groupCF":
+                holder.type = "groupCustomField";       // Overwrite the type since the Jira Cloud response has this.
+                holder.parameter = islp.getParameter(); // custom field name
+                break;
+              case "anyone":
+              case "sd.customer.portal.only":
+              default:
+                logger.warn("Unsupported issue security level permission type: " + islp.getType());
```

Comment:
I think we should track these somehow. Can we return some sort of `unknown` value to task_handlers so we can track this? Once we install the plugin, it's unlikely we'll get to see these logs, so best to export whatever we need.

Reply:
all the permissions are now returned.